operators/pkg/tenant-controller/tenant_controller.go

```diff
@@ -144,27 +148,26 @@ func (r *TenantReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 		retrigErr = err
 	}
 
+	// Determine if the personal namespace should be deleted
+
+	nsName := fmt.Sprintf("tenant-%s", strings.ReplaceAll(tn.Name, ".", "-"))
+
+	// Test if namespace has been open for too long; check if it is ok to delete
+	keepNsOpen, err := r.checkNamespaceKeepAlive(ctx, &tn, nsName, r.TenantNSKeepAlive)
+	if err != nil {
+		klog.Errorf("Error in r.List: unable to capture instances in tenant namespace %s -> %s", nsName, err)
+		tnOpinternalErrors.WithLabelValues("tenant", "self-update").Inc()
```

Comment:
```suggestion
		tnOpinternalErrors.WithLabelValues("tenant", "cluster-resources").Inc()
```

Reply:
Resolved in
[a496b60](https://github.com/netgroup-polito/CrownLabs/pull/821/commits/a496b6093f700d527ef3dc592afe22f11873d969)